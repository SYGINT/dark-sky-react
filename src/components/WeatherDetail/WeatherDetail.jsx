import React from "react";

import "./WeatherDetail.scss";

function WeatherDetail({ name, value }) {
  return (
    <span className="weather-detail">
      <span className="weather-detail-name">{name}:</span> {value}
    </span>
  );
}

export default WeatherDetail;
