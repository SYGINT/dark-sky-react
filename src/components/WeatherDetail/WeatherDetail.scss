.weather-detail {
  margin-right: 6px;
  line-height: 1.5;
  font-size: 1.15rem;

  &:last-child {
    margin-right: 0;
  }
}

.weather-detail-name {
  font-weight: bold;
}
