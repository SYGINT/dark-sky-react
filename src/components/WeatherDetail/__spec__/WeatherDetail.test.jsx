import React from "react";
import { shallow } from "enzyme";

import WeatherDetail from "../";

it("renders without crashing", () => {
  const wrapper = shallow(<WeatherDetail name="Feels Like" value="35&deg;" />);
  expect(wrapper).toMatchSnapshot();
});
