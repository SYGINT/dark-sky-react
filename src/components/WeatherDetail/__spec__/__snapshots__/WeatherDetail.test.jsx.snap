// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<span
  className="weather-detail "
>
  <span
    className="weather-detail-name"
  >
    Feels Like
    :
  </span>
   
  35Â°
   
</span>
`;
